src/main/java/hu/iac/webshop/domain/Customer.java

```diff
@@ -1,11 +1,88 @@
 package hu.iac.webshop.domain;
 
+import com.fasterxml.jackson.annotation.JsonBackReference;
+import com.fasterxml.jackson.annotation.JsonManagedReference;
+
+import javax.persistence.*;
 import java.util.List;
 
-public class Customer extends Account {
```

Comment:
Klopt het wel dat Customer niet Account extend?

Reply:
Is gecheckt, (nog) geen account aanwezig